lib/util/insightspublisher/index.js

```diff
@@ -111,7 +130,10 @@ class InsightsPublisher extends EventEmitter {
    * @returns {boolean} true if queued or published, false if disconnect() called
    */
   publish(groupName, eventName, payload) {
-    if (this._ws.readyState === this._WebSocket.CLOSING || this._ws.readyState === this._WebSocket.CLOSED) {
+    if (
```

Comment:
@makarandp0 Nitpick: can you refactor so that the operators are at the beginning of the line:
```js
if (this._ws !== null
  && (this._ws.readyState === this._WebSocket.CLOSING
    || this._ws.readyState === this._WebSocket.CLOSED))
```
?